exp/views/study.py

```diff
@@ -194,16 +194,22 @@ def get_form_kwargs(self):
     # UserPassesTestMixin.get_test_func()
     test_func = user_can_edit_study
 
+    # Rely on form validation to make sure structure is valid JSON; don't load/dump so we
+    # preserve formatting and ordering.
+    #
     def get_initial(self):
         """
```

Comment:
Fix indentation + adhere to Google style docstrings

Reply:
Fixed a few docstrings here (not feeling super ambitious about taking on all the docstrings of classes I touched at the moment, but let me know if you meant all of them)

Reply:
no, just the classes/functions you touch is fine